File: Checker source code with regex generated/Factorial check/test/tc1/tes1.cpp
#include <cstdio>
#include <cstdlib>
#include <iostream>

/*
 * fungsi faktorial
 * param int n
 * return n!
 */
int fac(int n){
	if (n==0) return 1;
	return n*fac(n-1);
}

int main(){
	int a,b;
	// baca nilai a dan b
	scanf("%d%d",&a,&b);//comment wkwk
	// tulis nilai a + b
	printf("%d\n",a+b);
	int d = fac(a);
	printf("%d %d\n",a,d);
	return 0;
}

File: Checker source code with regex generated/Factorial check/test/tc1/tes1.pas
var a,b,d:integer;

(*
 * fungsi faktorial
 * param int n
 * return n!
 *)
function fac(n:integer):integer;
begin
	if (n=0) then
	begin
		fac := 1;
	end else
	begin
		fac := n*fac(n-1);
	end;
end;

begin
	// baca nilai a dan b
	readln(a,b); //comment wkwk
	// tulis nilai a + b
	writeln(a+b);
	{
		comment lagi
	}
	d := fac(a);
	writeln(a,' ',d);
end.
